xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -543,6 +539,12 @@ void CGUIWindowPVRGuide::GetViewTimelineItems(CFileItemList &items)
       RefreshTimelineItems();
     }
 
+    if (m_newTimeline != nullptr)
+    {
+      m_cachedTimeline = m_newTimeline;
+      m_newTimeline.reset();
```

Comment:
As this is most likely to "transfer ownership" perhaps a unique_ptr should be prefered and then std::move used instead? Or do we have more ptrs holding m_newTimeline or m_cachedTimeline?